crates/common/src/compile.rs

```diff
@@ -59,6 +61,9 @@ pub struct ProjectCompiler {
 
     /// Extra files to include, that are not necessarily in the project's source dir.
     files: Vec<PathBuf>,
+
+    /// Whether to compile with preprocessed cache for tests and scripts.
+    optimize_tests: bool,
```

Comment:
Perhaps it is easier to understand for users if it was named `cache_tests`, optimization is very broad and could lead to incorrect assumptions

Reply:
good point, will change

Reply:
changed in https://github.com/foundry-rs/foundry/pull/10010/commits/ae939852a8d811255cbe64860406c108904635d4